tests/unit/CMakeLists.txt

```diff
@@ -466,6 +466,11 @@ seastar_add_test (source_location
 seastar_add_test (shared_token_bucket
   SOURCES shared_token_bucket_test.cc)
 
+seastar_add_test (prometheus
+  SOURCES
+    prometheus_test.cc
+    loopback_socket.hh)
```

Comment:
This header isn't a source.

Reply:
Copy paste from httpd... I'll clean this up when I upstream

Reply:
Removed now with the fmt changes